tests/toolbox_image.sh

```diff
@@ -51,6 +113,10 @@ teardown()
         return
     fi
 
+    if [ -r "${STORAGE_DEVICE_FILE}" ]; then
```

Comment:
during cleanup, i think we can use -e or -f; the file does not have to be readable by the user. I don't even think we need to have write access to the file, we just need write access to the upper laying directory